did-method-spec.md

```diff
@@ -0,0 +1,112 @@
+# TangleID DID Method Specification
+
+## About
+
+The TangleID DID method specification conforms to the requirements specified in the DID specification currently published by the W3C Credentials Community Group. For more information about DIDs and DID method specifications, please see the [DID Primer](https://github.com/WebOfTrustInfo/rwot5-boston/blob/master/topics-and-advance-readings/did-primer.md) and [DID Spec](https://w3c-ccg.github.io/did-spec/).
+
+## Abstraction
+
+The Tangle, IOTA’s consensus protocol, is a public distributed ledger with no transaction fee and quantum resistance. With these properties, we propose a DID method, TangleID, on this Distributed Ledger Technology. TangleID is optimized for DIDs and DID Documents.
+
+![](https://i.imgur.com/N8XIr6s.png)
+
+TangleID also optimizes MAM for key management and related features across the Tangle. Masked Authenticated Messaging (MAM) is a data communication protocol which adds functionality to broadcast and access data streams over the Tangle which adds integrity to these message streams. The owner of seed in MAM is able to create a channel structure like above to transfer the messages. TangleID stores and manages corresponding DID Documents on the MAM channels, and use the initial channel id as the DID’s `idstring`, each revision of the DID document is recorded on the message of the endpoint afterward.
```

Comment:
Rewrite the first sentence as:
> IOTA is a public distributed ledger that utilizes an invention called the Tangle at its core, address scalability issues and having no transaction fee, that encourages adoption of the technology in the industry.

Reply:
I added this sentence to the first paragraph. 👍